app/assets/javascripts/angular/angular-config.js.erb

```diff
@@ -7,3 +7,9 @@ app.config(['$routeProvider',function($routeProvider){
   })
 }])
 
+app.config(['$routeProvider',function($routeProvider){
+  $routeProvider.when('/pin/', {
+    templateUrl: "<%= asset_path('pin-show.html') %>",
+    controller: 'PinCtrl',
```

Comment:
Even if there's nothing in the controller, you still need to define it. Or, you can omit the controller key. For example,

```javascript

app.config(['$routeProvider',function($routeProvider){
  $routeProvider.when('/pin/', {
    templateUrl: "<%= asset_path('pin-show.html') %>"
  })
}])
```